python/tank/util/local_file_storage.py

```diff
@@ -235,7 +235,9 @@ def get_site_root(cls, hostname, path_type, generation=CORE_V18):
             # mysite.shotgunstudio.com -> mysite
             # shotgun.internal.int     -> shotgun.internal.int
             #
-            base_url = base_url.replace(".shotgunstudio.com", "")
+            base_url = base_url.replace(".shotgunstudio.com", "").replace(
+                ".shotgrid.autodesk.com", ""
```

Comment:
This doesn't match the other URLs. Is it correct? Should we strip out `.shotgridstudio.com` as well?

Reply:
Yeah, those were mistake. This is the real thing.